src/components/todoItem/TodoItem.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Do not to disable linter, try to fix this

Reply:
unfortunately could not fix this thing. But I want to say that these linter shutdowns were proposed in the condition for the task.